README.md

```diff
@@ -43,6 +43,15 @@ Also, non-square models can be loaded, but care should be taken to ensure that e
 dimension is an integer multiple of the filter size of the first conv layer (typically 16 or 32).
 
 
+### CPU allocations management
+
+As of ObjectDetector v0.3.4 on CPU an `AllocArrays` & `Adapt` - based allocator is used to reduce allocations.
```

Comment:
See my above comment, but if you do plan on bumping the version in this PR then you will need to also fix this version number.